lib/ansible/modules/clustering/kubevirt/kubevirt_raw.py

```diff
@@ -248,10 +248,5 @@ def _use_cdi(self, annotations, labels):
         return endpoint and app_label == 'containerized-data-importer'
 
 
-def main():
```

Comment:
It seems like keeping main method as entry point is a common [practice](https://github.com/kubernetes-incubator/kubespray/blob/master/library/kube.py#L313). Please only change line 253